examples/00-mapdl-examples/torsional_load.py

```diff
@@ -0,0 +1,201 @@
+mapdl = launch_mapdl(run_location=os.getcwd())
+
+
+##############################################################################
+# Define cylinder and mesh parameters
+
+torque = 100
+radius = 2
+h_tip = 2
+height = 20
+elemsize = 0.5
+pi = np.arccos(-1)
+force = 100 / radius
+pressure = force / (h_tip * 2 * np.pi * radius)
+
+##############################################################################
+# Model creation
+# --------------
+
+# Define higher-order SOLID186
+# Define surface effect elements SURF154 to apply torque
+# as a tangential pressure
+mapdl.prep7()
+mapdl.et(1, 186)
+mapdl.et(2, 154)
+mapdl.r(1)
+mapdl.r(2)
+
+##############################################################################
+# Aluminum properties (or something)
+mapdl.mp("ex", 1, 10e6)
+mapdl.mp("nuxy", 1, 0.3)
+mapdl.mp("dens", 1, 0.1 / 386.1)
+mapdl.mp("dens", 2, 0)
+
+# Simple cylinder
+for i in range(4):
+    mapdl.cylind(radius, "", "", height, 90 * (i - 1), 90 * i)
+
+mapdl.nummrg("kp")
+
+# interactive volume plot (optional)
+mapdl.vplot()
+
+
+##############################################################################
+# Mesh cylinder
+mapdl.lsel("s", "loc", "x", 0)
+mapdl.lsel("r", "loc", "y", 0)
+mapdl.lsel("r", "loc", "z", 0, height - h_tip)
+mapdl.lesize("all", elemsize * 2)
+mapdl.mshape(0)
+mapdl.mshkey(1)
+mapdl.esize(elemsize)
+mapdl.allsel("all")
+mapdl.vsweep("ALL")
+mapdl.csys(1)
+mapdl.asel("s", "loc", "z", "", height - h_tip + 0.0001)
+mapdl.asel("r", "loc", "x", radius)
+mapdl.local(11, 1)
+mapdl.csys(0)
+mapdl.aatt(2, 2, 2, 11)
+mapdl.amesh("all")
+mapdl.finish()
+
+# plot elements
+mapdl.eplot()
+
+##############################################################################
+# Solution
+# --------
+
+mapdl.slashsolu()  # Using Slash instead of / due to duplicate SOLU command
+mapdl.antype("static", "new")
+mapdl.eqslv("pcg", 1e-8)
+
+# Apply tangential pressure
+mapdl.esel("s", "type", "", 2)
+mapdl.sfe("all", 2, "pres", "", pressure)
+
+# Constrain bottom of cylinder/rod
+mapdl.asel("s", "loc", "z", 0)
+mapdl.nsla("s", 1)
+
+mapdl.d("all", "all")
+mapdl.allsel()
+mapdl.psf("pres", "", 2)
+mapdl.pbc("u", 1)
+mapdl.solve()
+
+
+##############################################################################
+# Post-processing
+# ---------------
+
+# You can access and plot the results within Python using PyMAPDL
+# with the following commands:
+
+# access the result from the mapdl result
+result = mapdl.result
+
+# Alternatively, open the result file using the path used in MAPDL
+# from ansys.mapdl import reader as pymapdl_reader
+# resultfile = os.path.join(mapdl.path, 'file.rst')
+# result = pymapdl_reader.read_binary(resultfile)
+
+##############################################################################
+# Access element results as arrays
+nnum, stress = result.nodal_stress(0)
+element_stress, elemnum, enode = result.element_stress(0)
+nodenum, stress = result.nodal_stress(0)
```

Comment:
Same, can we use `post_processing`?

If a method is missing, let me know, we can open an issue with that method, reference this issue, and merge this one. Later we will work in the other one.